packages/warriorjs-tower-beginner/src/index.js

```diff
@@ -216,7 +216,7 @@ export default {
     {
       description: 'You hear cries for help. Captives must need rescuing.',
       tip:
-        "Use `warrior.feel().isCaptive()` to see if there's a captive and `warrior.rescue()` to rescue him. Don't attack captives.",
+        "Use `warrior.feel().isFriendly()` to see if there's a captive and `warrior.rescue()` to rescue him. Don't attack captives.",
```

Comment:
`.isFriendly()` is not technically equivalent to the former `.isCaptive()` (although in practice it is) so we should recommend a safer approach to detect a captive unit: `warrior.feel().isFriendly() && warrior.feel().isBound()`.